tests/common/AppDelegate.cs

```diff
@@ -12,6 +12,9 @@
 using NUnit.Framework;
 using NUnit.Framework.Internal;
 
+// Disable until we get around to enable + fix any issues.
```

Comment:
Is there an issue for these?

Reply:
Basically https://github.com/xamarin/xamarin-macios/issues/6154